### ISSUE TITLE
TextClip

### ISSUE THREAD

**Issue by IDayday** [Zulko/moviepy#1285]
I install the moviepy by "pip install moviepy" , and moviepy=1.0.3 installed successfully.
When I test the TextClip function, I want to paint  "hello" on the video.
But I only get some short red lines like "___  ___  ___ " which are hardly noticeable.

my code:

```
from moviepy.editor import *

video = VideoFileClip("demo/write.mp4").subclip(2,6)

text_clip = ( TextClip("hello",fontsize=200,color='red')
             .set_position('center')
             .set_duration(4) )

result = CompositeVideoClip([video, txt_clip]) 
result.write_videofile("myHolidays_edited.mp4",fps=25)
```

**Comment by IDayday** [comment]
My environment:
lunix
python 3.6
moviepy 1.0.3
ImageMagick 7.0.10

I deployed the program on the server, so I don't have root permission.
I installed ImageMagick from source.
I have also checked the "policy.xml" , and I export the ImageMagick in PATH , also modify the "config_defaults.py"
IMAGEMAGICK_BINARY = r'/dat01/liuzhixiong/zt/imagemagick/bin/magick'

**Comment by keikoro** [comment]
(formatted code block)

**Comment by tburrows13** [comment]
See #1472 - TextClip's are being re-implemented in v2.0 so this will no longer be an issue.
